client/src/components/Dashboard/Dashboard.js

```diff
@@ -0,0 +1,39 @@
+import React, { Component } from 'react';
+import { withRouter } from 'react-router-dom';
+import { ToastContainer, toast } from 'react-toastify';
+
+class Dashboard extends Component {
+
```

Comment:
Made the dashboard a class component and when redirected to the dashboard via finishing the workflow, we'll toast the user, telling them "You have completed the setup!"